crates/storage/db-api/src/cursor.rs

```diff
@@ -75,6 +75,7 @@ pub trait DbDupCursorRO<T: DupSort> {
     ///
     /// # Note
     ///
+    /// The key must match exactly, while the subkey can be greater than or equal to.
```

Comment:
This is correct, cross-checked with MDBX docs https://libmdbx.dqdkfa.ru/group__c__cursors.html#gga501baff3bcff7e8e7818b4d412a2a6d5a2de3e789c951f80dbc05cc9c0fba82e8